common/build.gradle

```diff
@@ -107,6 +107,9 @@ dependencies {
     androidTestImplementation('androidx.test.espresso:espresso-core:3.3.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.google.guava:guava:29.0-android'
+    implementation "org.slf4j:slf4j-api:$slf4jVersion"
```

Comment:
This is required because DashJ 0.18 which is built by gradle instead of maven is not built as a bundle that includes all of its dependencies.

Though the kotlin lib is not used by DashJ.